src/controls/inputs-lifecycle.js

```diff
@@ -0,0 +1,68 @@
+var Crafty = require('../core/core.js'),
+    document = window.document;
+
+// figure out which eventName to listen to for mousewheel events
+var mouseWheelEvent = typeof document.onwheel !== 'undefined' ? 'wheel' : // modern browsers
+                        typeof document.onmousewheel !== 'undefined' ? 'mousewheel' : // old Webkit and IE
+                        'DOMMouseScroll'; // old Firefox
+
+//initialize the input events onload
+Crafty._preBind("Load", function () {
```

Comment:
For reference, prebind was added in #960, which provides some more context as to why it was necessary.